src/main/java/ch/admin/bag/covidcertificate/api/mapper/IssuableRapidTestMapper.java

```diff
@@ -0,0 +1,24 @@
+package ch.admin.bag.covidcertificate.api.mapper;
+
+import ch.admin.bag.covidcertificate.api.valueset.IssuableTestDto;
+import ch.admin.bag.covidcertificate.api.valueset.TestType;
+import ch.admin.bag.covidcertificate.domain.RapidTest;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class IssuableRapidTestMapper {
+    public static IssuableTestDto fromRapidTest(RapidTest rapidTest) {
+        return new IssuableTestDto(rapidTest.getCode(), rapidTest.getDisplay(), TestType.RAPID_TEST);
```

Comment:
Why do you add the test type here?

Reply:
This is a open question for me. I checked the Dto Object and there is a protected NoArgsConstructor and a AllArgsConstructor. Do we need to keep it null as RapidTests exist paralell to Tests in the DB?

Reply:
Oha, now I see it. The TestType is marked as JsonIgnore. I'll change it.